packages/core/src/modules/dids/domain/key-type/keyDidMapping.ts

```diff
@@ -24,6 +24,11 @@ const keyDidMapping: Record<KeyType, KeyDidMapping> = {
   [KeyType.Bls12381g1g2]: keyDidBls12381g1g2,
 }
 
+export const proofTypeKeyTypeMapping: Record<string, string> = {
+  Ed25519Signature2018: VERIFICATION_METHOD_TYPE_ED25519_VERIFICATION_KEY_2018,
+  BbsBlsSignature2020: VERIFICATION_METHOD_TYPE_BLS12381G2_KEY_2020,
+}
```

Comment:
There is a `getVerificationMethodTypesByProofType` method on the SignatureSuiteRegistry/W3cCredentialService available. Can we leverage that over this? Otherwise dynamically adding new signature suites won't work

Reply:
SignCredential takes a single verification method but the above method returns an array of verification method.

I will just use the first one found for now. Is that ok?